lib/sablon/content.rb

```diff
@@ -196,6 +197,22 @@ def initialize(source, attributes = {})
         #
         super name, img_data
         @attributes = attributes
+
+        def width
```

Comment:
Just noticed the width and height methods were inside the `initialize` method, easy fix to move them outside of it. That explains a lot of the warnings in the Travis CI log that had me confused.

Reply:
Thanks! That warning messages really had me confused as well and I didn't find a way to get rid of them...